README.md

```diff
@@ -1,3 +1,4 @@
+npm install @docusaurus/fasternpm install @docusaurus/core@3.6.3 @docusaurus/preset-classic@3.6.3
```

Comment:
think you accidentally copy + pasted here? 😅

Reply:
Actually it did it by itself, is was also amazed that this was happend. I thought maybe this is the new way of doing it hahahaa